app/code/community/Nexcessnet/Turpentine/Model/Observer/Esi.php

```diff
@@ -32,9 +32,9 @@ public function setCustomerGroupCookie( $eventObject ) {
         $customer = $eventObject->getCustomer();
         $cookie = Mage::getSingleton('core/cookie');
         if (Mage::getStoreConfig('persistent/options/enabled')) {
-            $cookie->set('customer_group', $customer->getGroupId(), Mage::getStoreConfig('persistent/options/lifetime'));
+            $cookie->set('customer_group', $customer->getGroupId(), Mage::getStoreConfig('persistent/options/lifetime'), '/');
         } else {
-            $cookie->set('customer_group', $customer->getGroupId());
+            $cookie->set('customer_group', $customer->getGroupId(), null, '/');
```

Comment:
This will overwrite every cookie path/domain no matter if it has a path/domain defined in the backend or not. is this on purpose @Stasevi4?